handbook/engineering/security/index.md

```diff
@@ -4,50 +4,39 @@ We think that security is an enabler for the business. Sourcegraph is committed
 
 ## Goals
 
-### Long term
+Our current work is documented in our [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fsecurity+is%3Aopen).
 
-Every organization is confident they can trust Sourcegraph cloud to securely store and manage their source code.
+### Visibility into Sourcegraph Cloud's attack surface
 
-### Medium term
+**Problem:** The Cloud team is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Before we can have [confidence in our security model](#confidence-in-our-security-model), we first need to have visibility into current risks, issues, and threats, so we can proactively plan what work needs to be done (i.e., we don't want to be relying on or waiting for security researchers or customer to report vulnerabilities to us).
 
-The goals below lie on the path to our long-term goals, and represent the next 3-6 months of work.  These goals will be delivered serially, but may be worked on in parallel.
+**Milestones:**
 
-**We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
+1. All Docker images are continuously scanned for known security vulnerabilities and the security team is alerted any time one is found.
+1. All compute nodes are continuously scanned for known security vulnerabilities and the security team is alerted any time one is found.
+   - GCP doesn't handle this?
+1. All high risk (I mean this informally, is there a better term of art to use here?) vulnerabilities found in scans are resolved.
+1. Implement centralized security logging.
+   - What is this? Is this just taking all our existing logs and shipping them to one place? Are there new kinds of logs we will want to add to our application? If so, what/where? We are already using LogDNA, is that a solution to this?
+1. Centralize workload and cloud (i.e. GCE) logs.
+   - Needs to be explained more.
+1. Normalize log format across services so it is easier to correlate and search, and this is a precursor to creating automated alerts from the logs.
```

Comment:
Normalize log format across services so that it is easier to correlate and search; this is a precursor to creating automated alerts from the logs.

Reply:
```suggestion
1. Normalize log format across services so that it is easier to correlate and search; this is a precursor to creating automated alerts from the logs.
```